tests/acceptance/features/apiSearchContent/propfindExtractedProps.feature

```diff
@@ -40,3 +42,84 @@ Feature: propfind extracted props
+    When user "Alice" gets the following extracted properties of resource "testaudio.mp3" inside space "Personal" using the WebDAV API
+      | propertyName |
+      | oc:audio     |
+    Then the HTTP status code should be "207"
+    And the "PROPFIND" response to user "Alice" should contain a mountpoint "testaudio.mp3" with these key and value pairs:
+      | key                | value                          |
+      | oc:audio/oc:album  | ALBUM1234567890123456789012345 |
+      | oc:audio/oc:artist | ARTIST123456789012345678901234 |
+      | oc:audio/oc:genre  | Pop                            |
+      | oc:audio/oc:title  | TITLE1234567890123456789012345 |
+      | oc:audio/oc:track  | 1                              |
+      | oc:audio/oc:year   | 2001                           |
+    When user "Alice" gets the following extracted properties of resource "testavatar.jpg" inside space "Personal" using the WebDAV API
+      | propertyName |
+      | oc:image     |
+      | oc:location  |
+      | oc:photo     |
+    Then the HTTP status code should be "207"
+    And the "PROPFIND" response to user "Alice" should contain a mountpoint "testavatar.jpg" with these key and value pairs:
+      | key                              | value                |
+      | oc:image/oc:width                | 640                  |
+      | oc:image/oc:height               | 480                  |
+      | oc:location/oc:latitude          | 43.467157            |
+      | oc:location/oc:longitude         | 11.885395            |
+      | oc:photo/oc:camera-make          | NIKON                |
+      | oc:photo/oc:camera-model         | COOLPIX P6000        |
+      | oc:photo/oc:f-number             | 4.5                  |
+      | oc:photo/oc:focal-length         | 6                    |
+
+
+  Scenario: check extracted properties of a file by sharee from shares space
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Brian    |
+    And user "Alice" has uploaded file "filesForUpload/testaudio.mp3" to "testaudio.mp3"
+    And user "Alice" has uploaded file "filesForUpload/testavatar.jpg" to "testavatar.jpg"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | testaudio.mp3 |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Viewer        |
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | testavatar.jpg |
+      | space           | Personal       |
+      | sharee          | Brian          |
+      | shareType       | user           |
+      | permissionsRole | Viewer         |
+    When user "Brian" gets the following extracted properties of resource "testaudio.mp3" inside space "Shares" using the WebDAV API
```

Comment:
Please, assert that users have respective shares. This prevents 404 when trying to list shares. we have this step to do so
```feature
And user "<user>" has a share "<resource>" synced
```